topics/metagenomics/tutorials/metagenomics-binning/tutorial.md

```diff
@@ -0,0 +1,385 @@
+
+As explained before, there are many challenges to metagenomics binning. The most common of them are listed below:
+
+- High complexity.
+- Fragmented sequences.
+- Uneven coverage.
+- Incomplete or partial genomes.
+- Horizontal gene transfer.
+- Chimeric sequences.
+- Strain variation.
+
+![Image show the binning process where sequences are grouped together based on genome signatures like the kmer profiles of each contig, contig coverage, or GC content](./binning.png "Binning"){:width="60%"}
+
+In this tutorial we will learn how to use **MetaBAT 2** tool through Galaxy. **MetaBAT** stands for "Metagenome Binning based on Abundance and Tetranucleotide frequency". It is:
+
+> *Grouping large fragments assembled from shotgun metagenomic sequences to deconvolute complex microbial communities, or metagenome binning, enables the study of individual organisms and their interactions. Here we developed automated metagenome binning software, called MetaBAT, which integrates empirical probabilistic distances of genome abundance and tetranucleotide frequency. On synthetic datasets MetaBAT on average achieves 98percent precision and 90% recall at the strain level with 281 near complete unique genomes. Applying MetaBAT to a human gut microbiome data set we recovered 176 genome bins with 92% precision and 80% recall. Further analyses suggest MetaBAT is able to recover genome fragments missed in reference genomes up to 19%, while 53 genome bins are novel. In summary, we believe MetaBAT is a powerful tool to facilitate comprehensive understanding of complex microbial communities ({%cite Kang2019%}).*
+
+We will use the uploaded assembled fasta files as input to the algorithm (For simplicity reasons all other parameters will be preserved with their default values).
+
+> <hands-on-title>Individual binning of short-reads with MetaBAT 2</hands-on-title>
+> 1.  {% tool [MetaBAT 2](toolshed.g2.bx.psu.edu/repos/iuc/megahit/megahit/1.2.9+galaxy0) %} with parameters:
+>     - *"Fasta file containing contigs"*: `assembly fasta files`
+<!-- >     - In *Advanced options*
+>       - *"Percentage of good contigs considered for binning decided by connection among contigs"*: `default 95`
+>       - *"Minimum score of an edge for binning"*: `default 60`
+>       - *"Maximum number of edges per node"*: `default 200`
+>       - *"TNF probability cutoff for building TNF graph"*: `default 0`
+>       - *"Turn off additional binning for lost or small contigs?"*: `default "No"`
+>       - *"Minimum mean coverage of a contig in each library for binning"*: `default 1`
+>       - *"Minimum total effective mean coverage of a contig for binning "*: `default 1`
+>       - *"For exact reproducibility"*: `default 0`
+>     - In *Output options*
+>       - *"Minimum size of a bin as the output"*: `default 200000`
+>       - *"Output only sequence labels as a list in a column without sequences?"*: `default "No"`
+>       - *"Save cluster memberships as a matrix format?"*: `"Yes"` -->
```

Comment:
Do these variables not need to be set?

Reply:
These variables can be accessed in Galaxy as advanced options for the algorithm. However @bebatut suggested to keep them hidden in order to keep the tutorial as simple as possible.